spec/gnarails/cli/application_spec.rb

```diff
@@ -12,7 +12,7 @@ module Cli
           application.options = { webpack: "react", skip_yarn: false, skip_git: true }
           application.new("name")
 
-          default_options = "-m /Users/jarvis/Code/Gnar/gnarails/lib/../gnarly.rb --asset-pipeline=propshaft --skip-test-unit --css=tailwind --javascript=esbuild --database=postgresql"
+          default_options = "-m /Users/jarvis/Code/Gnar/gnarails/lib/../gnarly.rb --asset-pipeline=propshaft --skip-test-unit --css=scss --javascript=esbuild --database=postgresql"
```

Comment:
Same here re: the path; do we never use the default options? Looks like it's been this way historically.

Reply:
This is handled properly in the latest commit